src/library/Ora/ReadModel/Task.php

```diff
@@ -32,6 +36,7 @@ class Task extends EditableEntity
     
     CONST ROLE_MEMBER = 'member';
     CONST ROLE_OWNER  = 'owner';
+    CONST NOT_MEMBER = 'notmember';
```

Comment:
Il metodo getMemberRole potrebbe essere chiamato anche con uno user che non è membro del Task. La costante l'ho introdotta solamente per far ritornare un valore diverso da NULL o da stringa vuota al metodo, in modo da coprire anche questo caso limite.